README.md

```
@@ -231,3 +231,77 @@ If using M1-based MacBook  run this before building image
+
+# Enable for a specific organization
+python manage.py manage_feature_flags --enable DATA_QUALITY --org-slug org-slug
+
+# Disable for a specific organization
+python manage.py manage_feature_flags --disable DATA_QUALITY --org-slug org-slug
+```
+
+#### 4. API Endpoint
+Frontend applications can fetch the feature flags for the current organization:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)